src/components/onCall/oncall-paid-status-dialog.tsx

```diff
@@ -88,20 +114,27 @@ const OnCallHandler = ({
             />
           </Typography>
           <Typography variant="subtitle1">
-            <b>Date:</b> {weekRange}
+            <Typography component="span" fontWeight="bold" display="inline">
+              {strings.oncall.date}:
+            </Typography> {weekRange}
```

Comment:
Same here, not sure what the reason is for nested typography but it can probably be avoided with appropriate styling.